packages/headless/src/features/insight-search/insight-search-actions.ts

```diff
@@ -303,6 +306,7 @@ const buildInsightSearchRequest = (
       firstResult: state.pagination.firstResult,
       numberOfResults: state.pagination.numberOfResults,
     }),
+    ...(cq && {cq}),
```

Comment:
BTW I don't know how necessary it is, but in the current search request to the search api, we do send the 
`tab: tabName` as part of the payload. 
If you are including the `cq` of the tab maybe it makes sense to also include that?

Reply:
Good point!  I see that the tab parameter is used mainly for analytics purposes since it represents the originLevel2.
This will need to be added as well, I will create a Jira for it and address it in a separate PR just to avoid blocking the UI-KIT release any longer.

Reply:
Good idea 👍